src/main/java/marquez/core/exceptions/DaoException.java

```diff
@@ -0,0 +1,3 @@
+package marquez.core.exceptions;
+
+public class DaoException extends Exception {}
```

Comment:
Is this exception being used?

Reply:
Not yet, but I put it in to represent errors that occur at the DAO layer. We can also remove it for this review if we think that's better.

Reply:
Yeah, I would remove `DaoException` as it wouldn't represent the exceptions that might/would be thrown. We'd mainly be handling `SqlException` and therefore wouldn't have to define domain exceptions.